crates/router/src/core/payments/transformers.rs

```diff
@@ -348,6 +364,7 @@ where
                                 .map(ForeignInto::foreign_into),
                         )
                         .set_metadata(payment_intent.metadata)
+                        .set_connector_label(connector_label)
```

Comment:
Here I think it's better if we also return the `business_country`, `business_label`, `business_sublabel` and `connector_name` separately alongside the `connector_label`. Will make accessing that data easier downstream as opposed to deconstructing the `connector_label` to get it.

Reply:
These will be 3 additional fields. We can send it, but I felt like the response will contain too many fields. Thinking of it, better to send them in the response.